package.json

```diff
@@ -39,7 +39,7 @@
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
     "deep-equal": "1.0.1",
-    "faye": "smooch/faye#temp-smooch-fix",
+    "faye": "smooch/faye#faye-fixes",
```

Comment:
maybe it would be easier to target the commit sha instead of creating branches each time..?

Reply:
My thought was that from now on we'd use this branch. I wanted a place where I could open PRs to if we needed further fixes.

Reply:
Might be easier to trigger new deploys (and see what actually changed) if we change the sha each time though

Reply:
Sure